docs/HelloWorld.rst

```diff
@@ -238,67 +252,77 @@ The code:
     [info] Applying mutations (threads: 1)
-           [################################] 2/2. Finished in 0ms
+           [################################] 5/5. Finished in 0ms
     [info] Compiling original code (threads: 1)
-           [################################] 1/1. Finished in 12ms
-    [info] Running mutants (threads: 2)
-           [################################] 2/2. Finished in 10ms
-    [info] Killed mutants (2/2):
-    /tmp/sc-b3yQyijWP/main.cpp:2:11: warning: Killed: Replaced >= with > [cxx_ge_to_gt]
-      if (age >= 21) {
-              ^
-    /tmp/sc-b3yQyijWP/main.cpp:2:11: warning: Killed: Replaced >= with < [cxx_ge_to_lt]
-      if (age >= 21) {
-              ^
+           [################################] 1/1. Finished in 11ms
+    [info] Link mutated program (threads: 1)
+           [################################] 1/1. Finished in 62ms
+    [info] Warm up run (threads: 1)
+           [################################] 1/1. Finished in 311ms
+    [info] Baseline run (threads: 1)
+           [################################] 1/1. Finished in 19ms
+    [info] Running mutants (threads: 5)
+           [################################] 5/5. Finished in 63ms
+    [info] Killed mutants (5/5):
+    /tmp/sc-PzmaCNIRu/main.cpp:2:15: warning: Killed: Replaced >= with > [cxx_ge_to_gt]
+          if (age >= 21) {
+                  ^
+    /tmp/sc-PzmaCNIRu/main.cpp:2:15: warning: Killed: Replaced >= with < [cxx_ge_to_lt]
+          if (age >= 21) {
+                  ^
+    /tmp/sc-PzmaCNIRu/main.cpp:9:34: warning: Killed: Replaced == with != [cxx_eq_to_ne]
+          bool test1 = valid_age(25) == true;
+                                     ^
+    /tmp/sc-PzmaCNIRu/main.cpp:15:34: warning: Killed: Replaced == with != [cxx_eq_to_ne]
+          bool test2 = valid_age(20) == false;
+                                     ^
+    /tmp/sc-PzmaCNIRu/main.cpp:21:34: warning: Killed: Replaced == with != [cxx_eq_to_ne]
+          bool test3 = valid_age(21) == true;
+                                     ^
     [info] All mutations have been killed
     [info] Mutation score: 100%
+    [info] Total execution time: 554ms
+
+In this last run we see that all mutants were killed since we covered with tests
+all cases around the ``<=``.
 
 Summary
 -------
 
-This is a short summary of what we have learned in tutorial:
-
-- Your code has to be compiled with ``-fembed-bitcode -g`` compile flags:
+This is a short summary of what we have learned in tutorial.
+Your code has to be compiled with ``-fembed-bitcode -g`` compile flags:
 
   - Mull expects embedded bitcode files to be present in a binary executable
     (ensured by ``-fembed-bitcode``).
 
   - Mull needs debug information to be included by the compiler (enabled by
     ``-g``). Mull uses this information to find mutations in bitcode and source
     code.
-
-- Mull expects the following arguments to be always provided:
```

Comment:
Why did you remove this one?

Reply:
There were two of them before, now there is only one so I find it weird to have this paragraph here. Shall I put it back?

Reply:
I understand now. I don't think it is weird, it just says the truth - you only need to provide an executable. Use your best judgement 👍